app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveViewModel.java

```diff
@@ -482,4 +1052,42 @@ public void setCompensationToTime(String compensationToTime) {
     public User getUser() {
         return mUser;
     }
+
+    @IntDef({
+            CurrentTimeSelected.NONE, CurrentTimeSelected.CHECK_IN, CurrentTimeSelected.CHECK_OUT,
+            CurrentTimeSelected.COMPENSATION_FROM, CurrentTimeSelected.COMPENSATION_TO
+    })
+    @interface CurrentTimeSelected {
+        int NONE = 0;
+        int CHECK_IN = 1;
+        int CHECK_OUT = 2;
+        int COMPENSATION_FROM = 3;
+        int COMPENSATION_TO = 4;
+    }
+
+    @StringDef({
+            LeaveType.IN_LATE_M, LeaveType.IN_LATE_A, LeaveType.LEAVE_EARLY_M,
+            LeaveType.LEAVE_EARLY_A, LeaveType.LEAVE_OUT, LeaveType.FORGOT_CHECK_ALL_DAY,
+            LeaveType.FORGOT_TO_CHECK_IN, LeaveType.FORGOT_TO_CHECK_OUT,
+            LeaveType.FORGOT_CARD_ALL_DAY, LeaveType.FORGOT_CARD_IN, LeaveType.FORGOT_CARD_OUT,
+            LeaveType.IN_LATE_WOMAN_M, LeaveType.IN_LATE_WOMAN_A, LeaveType.LEAVE_EARLY_WOMAN_M,
+            LeaveType.LEAVE_EARLY_WOMAN_A
+    })
+    public @interface LeaveType {
+        String IN_LATE_M = "In late (M)";
+        String IN_LATE_A = "In late (A)";
+        String LEAVE_EARLY_M = "Leave early (M)";
+        String LEAVE_EARLY_A = "Leave early (A)";
+        String LEAVE_OUT = "Leave out";
+        String FORGOT_CHECK_ALL_DAY = "Forgot to check in/check out (all day)";
+        String FORGOT_TO_CHECK_IN = "Forgot to check in";
+        String FORGOT_TO_CHECK_OUT = "Forgot to check out";
+        String FORGOT_CARD_ALL_DAY = "Forgot card (all day)";
+        String FORGOT_CARD_IN = "Forgot card (in)";
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.